screens/DaterProfile.tsx

```
@@ -0,0 +1,239 @@
+  name: string;
+  link: string;
+  rankScore: number;
+}
+
+interface FriendsProps {
+  name: string;
+  vestedScore: number;
+}
+
+export default function DaterProfile() {
+  const [profile, setProfile] = useState<DaterProfileProps>({
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(@stylistic/js/indent)